strr-web/components/bcros/status-card/StatusCard.vue

```diff
@@ -97,4 +86,10 @@ const isApproved = computed(() => {
     ].includes(hostStatus)
   }
 })
+const showDownloadButton = computed(() => isRegistrationActive && canDownloadCertificate.value)
+onMounted(async () => {
+  canDownloadCertificate.value = registrationId
+    ? await isCertificateIssued(registrationId.toString())
```

Comment:
I'm not a fan of the idea of calling API for each card that has the `registrationId`, just to show the Download Certificate button.
Is it because the registration can exist without the certificate because the Examiner has not issued it yet?

Reply:
This has been reworked and now relying on `isCertificateIssued` in the application header provided by backend.